packages/types/js.ts

```diff
@@ -33,6 +33,7 @@ export const ZJsAppStateSync = z.object({
   userId: z.string().optional(),
   surveys: z.union([z.array(ZSurvey), z.array(ZLegacySurvey)]),
   noCodeActionClasses: z.array(ZActionClass),
+  actionClasses: z.array(ZActionClass).optional(),
```

Comment:
This shouldn't be optimal in the current release. If this causes issues in the sync endpoint we should introduce a legacy type that has the fields and could also be returned from the sync endpoint.